src/index.html

```diff
@@ -160,6 +167,37 @@ <h2 class="modal__title">New post</h2>
         </div>
       </div>
 
+      <div id="avatar-modal" class="modal">
+        <div class="modal__container">
+          <h2 class="modal__title">Edit profile avatar</h2>
+          <button
+            type="button"
+            id="modal__close-button-avatar"
+            class="modal__close-button"
+          ></button>
+          <form name="add-avatar" id="add-avatar" class="modal__form">
+            <label for="image-link" class="modal__label">
+              Profile Picture
+              <input
+                id="image-link"
```

Comment:
`id="image-link"` os a;readu om ;ome 135, so you need to use another `id` and `form` for the  `avatar` modal



You can use `avatar-link`